File: Game of life/src/Organism.java
public class Organism {
   private boolean isLife = false;
   public void kill(){
      isLife = false;
   }
   public void resurrect(){
      isLife = true;

   }
   public void draw(){
      if (isLife==true){
         System.out.print("0");
      }else{
         System.out.print(" ");
      }
   }

   public boolean isLife() {
      return isLife;
   }
}

File: GameOfLifeCS/GameOfLifeCS/Organism.cs
public class Organism
{
    private bool _isLife = false;
    public void kill()
    {
        _isLife = false;
    }
    public void resurrect()
    {
        _isLife = true;

    }
    public void draw()
    {
        if (_isLife == true)
        {
            Console.Write("0");
        }
        else
        {
            Console.Write(" ");
        }
    }

    public bool isLife()
    {
        return _isLife;
    }
}